<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>381</width>
    <height>390</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>371</width>
    <height>293</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>381</width>
    <height>390</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Simbolos para la funcion</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>Icon.png</normaloff>Icon.png</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: #cfc4b1;</string>
  </property>
  <widget class="QLineEdit" name="funcLine">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>80</y>
     <width>221</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: #BAB3A0 ;
font-family: Times New Roman;
font-size: 24px;
border: 2px solid rgb(69, 85, 61);
border-radius: 15px;
</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>80</y>
     <width>121</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: #cfc4b1;
font-family: Times New Roman;
font-size: 24px;
border-bottom: 2px solid rgb(69, 85, 61);</string>
   </property>
   <property name="text">
    <string>F(variable) =</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="borrar">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>140</y>
     <width>111</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
background-color: #6d7e5f ;
font-family: Times New Roman;
font-size: 24px;
border: 2px solid #45553d;
border-radius: 10px;
color: white;
}

QPushButton:hover{
background-color: #333B2D;
border: 2px solid white;
}</string>
   </property>
   <property name="text">
    <string>AC</string>
   </property>
  </widget>
  <widget class="QPushButton" name="elevar">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>190</y>
     <width>51</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
background-color: #6d7e5f ;
font-family: Times New Roman;
font-size: 24px;
border: 2px solid #45553d;
border-radius: 10px;
color: white;
}

QPushButton:hover{
background-color: #333B2D;
border: 2px solid white;
}</string>
   </property>
   <property name="text">
    <string>a^x</string>
   </property>
  </widget>
  <widget class="QPushButton" name="raiz">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>240</y>
     <width>51</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
background-color: #6d7e5f ;
font-family: Times New Roman;
font-size: 24px;
border: 2px solid #45553d;
border-radius: 10px;
color: white;
}

QPushButton:hover{
background-color: #333B2D;
border: 2px solid white;
}</string>
   </property>
   <property name="text">
    <string>√</string>
   </property>
  </widget>
  <widget class="QPushButton" name="quitar">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>190</y>
     <width>51</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
background-color: #6d7e5f ;
font-family: Times New Roman;
font-size: 24px;
border: 2px solid #45553d;
border-radius: 10px;
color: white;
}

QPushButton:hover{
background-color: #333B2D;
border: 2px solid white;
}</string>
   </property>
   <property name="text">
    <string>⌫</string>
   </property>
  </widget>
  <widget class="QPushButton" name="division">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>190</y>
     <width>51</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
background-color: #6d7e5f ;
font-family: Times New Roman;
font-size: 24px;
border: 2px solid #45553d;
border-radius: 10px;
color: white;
}

QPushButton:hover{
background-color: #333B2D;
border: 2px solid white;
}</string>
   </property>
   <property name="text">
    <string>÷</string>
   </property>
  </widget>
  <widget class="QPushButton" name="multiplicacion">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>240</y>
     <width>51</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
background-color: #6d7e5f ;
font-family: Times New Roman;
font-size: 24px;
border: 2px solid #45553d;
border-radius: 10px;
color: white;
}

QPushButton:hover{
background-color: #333B2D;
border: 2px solid white;
}</string>
   </property>
   <property name="text">
    <string>×</string>
   </property>
  </widget>
  <widget class="QPushButton" name="menos">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>290</y>
     <width>51</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
background-color: #6d7e5f ;
font-family: Times New Roman;
font-size: 24px;
border: 2px solid #45553d;
border-radius: 10px;
color: white;
}

QPushButton:hover{
background-color: #333B2D;
border: 2px solid white;
}</string>
   </property>
   <property name="text">
    <string>-</string>
   </property>
  </widget>
  <widget class="QPushButton" name="proceder">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>140</y>
     <width>111</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
background-color: #6d7e5f ;
font-family: Times New Roman;
font-size: 28px;
border: 2px solid #45553d;
border-radius: 10px;
color: white;
}

QPushButton:hover{
background-color: #333B2D;
border: 2px solid white;
}</string>
   </property>
   <property name="text">
    <string>=</string>
   </property>
  </widget>
  <widget class="QPushButton" name="equis">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>290</y>
     <width>51</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
background-color: #6d7e5f ;
font-family: Times New Roman;
font-size: 24px;
border: 2px solid #45553d;
border-radius: 10px;
color: white;
}

QPushButton:hover{
background-color: #333B2D;
border: 2px solid white;
}</string>
   </property>
   <property name="text">
    <string>X</string>
   </property>
  </widget>
  <widget class="QPushButton" name="ye">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>240</y>
     <width>51</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
background-color: #6d7e5f ;
font-family: Times New Roman;
font-size: 24px;
border: 2px solid #45553d;
border-radius: 10px;
color: white;
}

QPushButton:hover{
background-color: #333B2D;
border: 2px solid white;
}</string>
   </property>
   <property name="text">
    <string>Y</string>
   </property>
  </widget>
  <widget class="QPushButton" name="mas">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>340</y>
     <width>51</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
background-color: #6d7e5f ;
font-family: Times New Roman;
font-size: 24px;
border: 2px solid #45553d;
border-radius: 10px;
color: white;
}

QPushButton:hover{
background-color: #333B2D;
border: 2px solid white;
}</string>
   </property>
   <property name="text">
    <string>+</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>371</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: #cfc4b1;
font-family: Times New Roman;
font-size: 24px;
border-bottom: 2px solid rgb(69, 85, 61);</string>
   </property>
   <property name="text">
    <string>Agrega signos espciales a la función:</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="diezElevado">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>290</y>
     <width>51</width>
     <height>41</height>
    </rect>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
background-color: #6d7e5f ;
font-family: Times New Roman;
font-size: 15px;
border: 2px solid #45553d;
border-radius: 10px;
color: white;
}

QPushButton:hover{
background-color: #333B2D;
border: 2px solid white;
}</string>
   </property>
   <property name="text">
    <string>x10^x</string>
   </property>
  </widget>
  <widget class="QPushButton" name="punto">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>340</y>
     <width>51</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
background-color: #6d7e5f ;
font-family: Times New Roman;
font-size: 24px;
border: 2px solid #45553d;
border-radius: 10px;
color: white;
}

QPushButton:hover{
background-color: #333B2D;
border: 2px solid white;
}</string>
   </property>
   <property name="text">
    <string>.</string>
   </property>
  </widget>
  <widget class="QPushButton" name="parentIzq">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>340</y>
     <width>51</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
background-color: #6d7e5f ;
font-family: Times New Roman;
font-size: 24px;
border: 2px solid #45553d;
border-radius: 10px;
color: white;
}

QPushButton:hover{
background-color: #333B2D;
border: 2px solid white;
}</string>
   </property>
   <property name="text">
    <string>(</string>
   </property>
  </widget>
  <widget class="QPushButton" name="parentDer">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>340</y>
     <width>51</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
background-color: #6d7e5f ;
font-family: Times New Roman;
font-size: 24px;
border: 2px solid #45553d;
border-radius: 10px;
color: white;
}

QPushButton:hover{
background-color: #333B2D;
border: 2px solid white;
}</string>
   </property>
   <property name="text">
    <string>)</string>
   </property>
  </widget>
  <widget class="QPushButton" name="exit">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>140</y>
     <width>111</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
background-color: #FF3B3B ;
font-family: Times New Roman;
font-size: 20px;
border: 2px solid #45553d;
border-radius: 10px;
color: white;
}

QPushButton:hover{
background-color: #AC0000;
border: 2px solid white;
}</string>
   </property>
   <property name="text">
    <string>EXIT</string>
   </property>
  </widget>
  <widget class="QPushButton" name="seis">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>240</y>
     <width>51</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
background-color: #6d7e5f ;
font-family: Times New Roman;
font-size: 24px;
border: 2px solid #45553d;
border-radius: 10px;
color: white;
}

QPushButton:hover{
background-color: #333B2D;
border: 2px solid white;
}</string>
   </property>
   <property name="text">
    <string>6</string>
   </property>
  </widget>
  <widget class="QPushButton" name="tres">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>290</y>
     <width>51</width>
     <height>41</height>
    </rect>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
background-color: #6d7e5f ;
font-family: Times New Roman;
font-size: 24px;
border: 2px solid #45553d;
border-radius: 10px;
color: white;
}

QPushButton:hover{
background-color: #333B2D;
border: 2px solid white;
}</string>
   </property>
   <property name="text">
    <string>3</string>
   </property>
  </widget>
  <widget class="QPushButton" name="cinco">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>240</y>
     <width>51</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
background-color: #6d7e5f ;
font-family: Times New Roman;
font-size: 24px;
border: 2px solid #45553d;
border-radius: 10px;
color: white;
}

QPushButton:hover{
background-color: #333B2D;
border: 2px solid white;
}</string>
   </property>
   <property name="text">
    <string>5</string>
   </property>
  </widget>
  <widget class="QPushButton" name="uno">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>290</y>
     <width>51</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
background-color: #6d7e5f ;
font-family: Times New Roman;
font-size: 24px;
border: 2px solid #45553d;
border-radius: 10px;
color: white;
}

QPushButton:hover{
background-color: #333B2D;
border: 2px solid white;
}</string>
   </property>
   <property name="text">
    <string>1</string>
   </property>
  </widget>
  <widget class="QPushButton" name="dos">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>290</y>
     <width>51</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
background-color: #6d7e5f ;
font-family: Times New Roman;
font-size: 24px;
border: 2px solid #45553d;
border-radius: 10px;
color: white;
}

QPushButton:hover{
background-color: #333B2D;
border: 2px solid white;
}</string>
   </property>
   <property name="text">
    <string>2</string>
   </property>
  </widget>
  <widget class="QPushButton" name="siete">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>190</y>
     <width>51</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
background-color: #6d7e5f ;
font-family: Times New Roman;
font-size: 24px;
border: 2px solid #45553d;
border-radius: 10px;
color: white;
}

QPushButton:hover{
background-color: #333B2D;
border: 2px solid white;
}</string>
   </property>
   <property name="text">
    <string>7</string>
   </property>
  </widget>
  <widget class="QPushButton" name="ocho">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>190</y>
     <width>51</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
background-color: #6d7e5f ;
font-family: Times New Roman;
font-size: 24px;
border: 2px solid #45553d;
border-radius: 10px;
color: white;
}

QPushButton:hover{
background-color: #333B2D;
border: 2px solid white;
}</string>
   </property>
   <property name="text">
    <string>8</string>
   </property>
  </widget>
  <widget class="QPushButton" name="nueve">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>190</y>
     <width>51</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
background-color: #6d7e5f ;
font-family: Times New Roman;
font-size: 24px;
border: 2px solid #45553d;
border-radius: 10px;
color: white;
}

QPushButton:hover{
background-color: #333B2D;
border: 2px solid white;
}</string>
   </property>
   <property name="text">
    <string>9</string>
   </property>
  </widget>
  <widget class="QPushButton" name="ceroBtn">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>340</y>
     <width>111</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
background-color: #6d7e5f ;
font-family: Times New Roman;
font-size: 24px;
border: 2px solid #45553d;
border-radius: 10px;
color: white;
}

QPushButton:hover{
background-color: #333B2D;
border: 2px solid white;
}</string>
   </property>
   <property name="text">
    <string>0</string>
   </property>
  </widget>
  <widget class="QPushButton" name="cuatro">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>240</y>
     <width>51</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
background-color: #6d7e5f ;
font-family: Times New Roman;
font-size: 24px;
border: 2px solid #45553d;
border-radius: 10px;
color: white;
}

QPushButton:hover{
background-color: #333B2D;
border: 2px solid white;
}</string>
   </property>
   <property name="text">
    <string>4</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
